content/collections/guides_and_surveys/en/sdk.md

```diff
@@ -49,7 +49,23 @@ amplitude.add(engagementPlugin());
 
 ### Third-party analytics provider
 
-Using the Guides and Surveys standalone SDK with another analytics provider requires extra configuration to help map properties to Amplitude. This initialization code accepts parameters that define the user and any integrations.
+Using the Guides and Surveys standalone SDK with another analytics provider requires extra configuration to help map properties to Amplitude.
+
+{{partial:collapse name="Usign NPM or Yarn"}}
+
+Calling `init` is only required when loading Guides and Surveys via NPM or Yarn. If you're using the script installation, skip straight to calling `boot`.
+
+```js
+engagement.init(apiKey: string, options: { serverZone: "US" | "EU" }): void
```

Comment:
I realized that we actually default `serverZone` to `local` when it's not passed. For that reason I decided to mark this as 'required', but we should change this to default to 'US'.